vmdb/app/models/mixins/web_server_worker_mixin.rb

```diff
@@ -69,7 +81,7 @@ def self.sync_workers
             port = self.reserve_port(ports)
             $log.info("MIQ(#{self.name}.sync_workers) Reserved port=#{port}, Current ports in use: #{ports.inspect}")
             ports << port
-            w = self.start_worker(:uri => "http://0.0.0.0:#{port}")
+            w = self.start_worker(:uri => build_uri(port))
```

Comment:
why not `"http://#{BINDING_ADDRESS}:#{port}"`. Are you thinking IPv6?

or better yet `"http://127.0.0.1:3000"` and get rid of all the parameterization